Remotion/Web/Core/UI/Controls/ControlItemCollection.cs

```diff
@@ -269,7 +270,7 @@ private bool IsSupportedType (IControlItem controlItem)
       }
     }
 
-    public void Dispatch (IDictionary values, Control? parent, string collectionName)
+    public void Dispatch (IDictionary<string, IDictionary<string, WebString>> values, Control? parent, string collectionName)
```

Comment:
Disclaimer: his is just an observation based on the current changeset, since actually changing the code would result in probably more work than is justified (unless you actually want to do this change):

It would be cool if we had these typed as IReadOnlyDictionary<string, IReadOnlyDictionary<string, WebString>> since there is no write access. But we would need to rewrite (or cast) at some points in the code, which, combined with the low priority if this feature doesn't really make this more than a pet-peeve.

Also, the aggressive use of IReadOnlyCollection/IReadOnlyDictionary during specificiation of a task is something that requires more Pavlovian conditioning...